recipes/pytest-black/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pytest-black" %}
+{% set version = "0.3.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 01a9a7acc69e618ebf3f834932a4d7a81909f6911051d0871b0ed4de3cbe9712
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - black >=19.3b0
+    - pip
+    - pytest >=3.5.0
+    - python
+    - setuptools
+    - setuptools_scm
+    - toml
+  run:
+    - black >=19.3b0
+    - pytest >=3.5.0
+    - python
+    - setuptools
+    - setuptools_scm
+    - toml
+
```

Please add the test section
Something like
```suggestion
test:
  imports:
    - pytest_black
  commands:
    - pip check
  requires:
    - pip
```